lib/http-proxy/passes/web-incoming.js

```diff
@@ -143,9 +148,18 @@ module.exports = {
     }
 
     // Ensure we abort proxy if request is aborted
-    req.on('aborted', function () {
-      proxyReq.abort();
-    });
+    if (supportsAbortedEvent) {
```

Comment:
`if (supportsAbortedEvent) { req.on('aborted', () => proxyReq.abort()); } else { res.on('close', () => { if (!res.writableFinished) proxyReq.abort(); }); }
`
This would be suitable too i think